monkey/common/utils/exceptions.py

```diff
@@ -20,3 +20,7 @@ class CredentialsNotRequiredError(RegistrationNotNeededError):
 
 class AlreadyRegisteredError(RegistrationNotNeededError):
     """ Raise to indicate the reason why registration is not required """
+
+
+class VersionServerConnectionError(Exception):
```

Comment:
Should be part of this PR?

Reply:
Probably not, but since this is a small refactoring on separate commits, I think it's fine